### ISSUE TITLE
Resolution time criteria not working

### ISSUE THREAD

**Issue by Hildy** [jstanden/cerb#691]
Attempting to use message->ticket->first->resolution as a criteria for a behavior outcome universally fails.  The debug statement in the attached behavior outputs what appears to be the correct value, but any evaluation of that value simply doesn't work.  It reports failure in the simulator and it also doesn't work live.

Behavior is
```
{
    "behavior": {
        "uid": "behavior_139",
        "title": "Comment if it's been closed before",
        "is_disabled": false,
        "is_private": false,
        "priority": 50,
        "event": {
            "key": "event.mail.received",
            "label": "New message added to ticket"
        },
        "nodes": [
            {
                "type": "action",
                "title": "Debug",
                "status": "live",
                "params": {
                    "actions": [
                        {
                            "action": "create_comment",
                            "on": "ticket_id",
                            "content": "We ran the behavior...\r\n\r\nResolution time is {{ticket_elapsed_resolution_first}}."
                        }
                    ]
                }
            },
            {
                "type": "switch",
                "title": "Was the ticket previously closed?",
                "status": "live",
                "nodes": [
                    {
                        "type": "outcome",
                        "title": "Yes",
                        "status": "live",
                        "params": {
                            "groups": [
                                {
                                    "any": 0,
                                    "conditions": [
                                        {
                                            "condition": "ticket_elapsed_resolution_first",
                                            "oper": "gt",
                                            "value": "0"
                                        }
                                    ]
                                }
                            ]
                        },
                        "nodes": [
                            {
                                "type": "action",
                                "title": "Comment on ticket",
                                "status": "live",
                                "params": {
                                    "actions": [
                                        {
                                            "action": "create_comment",
                                            "on": "ticket_id",
                                            "content": "Yes branch.\r\n\r\nResolution time is {{ticket_elapsed_resolution_first}}."
                                        }
                                    ]
                                }
                            }
                        ]
                    },
                    {
                        "type": "outcome",
                        "title": "No",
                        "status": "live",
                        "params": {
                            "groups": [
                                {
                                    "any": 0,
                                    "conditions": [
                                        {
                                            "condition": "ticket_elapsed_resolution_first",
                                            "oper": "is",
                                            "value": "0"
                                        }
                                    ]
                                }
                            ]
                        },
                        "nodes": [
                            {
                                "type": "action",
                                "title": "Comment on ticket",
                                "status": "live",
                                "params": {
                                    "actions": [
                                        {
                                            "action": "create_comment",
                                            "on": "ticket_id",
                                            "content": "No branch.\r\n\r\nResolution time is {{ticket_elapsed_resolution_first}}."
                                        }
                                    ]
                                }
                            }
                        ]
                    }
                ]
            }
        ]
    }
}
```

**Comment by jstanden** [comment]
I'll give this a test right now.

**Comment by jstanden** [comment]
I can reproduce the issue. Investigating.

**Comment by jstanden** [comment]
Fixed in 8.3.9
